engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -791,6 +176,11 @@ public float getExposure() {
         return currentExposure;
     }
 
+    // TODO: Remove this method, temporarly here for DownSampleSceneAndUpdateExposure
```

Comment:
This will be removed once, nodes can be queried from "RenderGraph" via their identifiers. 

_Yet another but much better solution for this is to move ShaderParameter classes inside nodes._